mpitests/test_apply.py

```diff
@@ -161,45 +119,6 @@ def circuit():
     assert np.allclose(local_state_vector, local_expected_output_cpu, atol=tol, rtol=0)
 
 
-def apply_operation_gates_apply_nonparam(tol, dev_mpi, operation, Wires):
```

Comment:
Are these tests moved elsewhere now?

Reply:
No, these functions only help to compute the result of the circuit but using the old API `dev.apply(ops)` then they were no longer needed. :rocket: